test/e2e/framework/util.go

```diff
@@ -474,8 +477,8 @@ func ProxyMode(f *Framework) (string, error) {
 			Containers: []v1.Container{
 				{
 					Name:    "detector",
-					Image:   imageutils.GetE2EImage(imageutils.Agnhost),
-					Command: []string{"/bin/sleep", "3600"},
+					Image:   AgnHostImage,
+					Command: []string{"pause"},
```

Comment:
The ```AgnHostImage``` has this [```pause```](https://github.com/kubernetes/kubernetes/tree/master/test/images/agnhost#pause) binary specifically for this.

Reply:
**pause**
It will pause the execution of the binary. This can be used for containers which have to be kept in a Running state for various purposes, including executing other agnhost commands.

Usage:

    kubectl exec test-agnhost -- /agnhost pause

Reply:
I got it, thanks for your explanation.